drivers/usb/device/usb_dc_sam0.c

```diff
@@ -424,6 +471,23 @@ int usb_dc_ep_enable(const u8_t ep)
 	return 0;
 }
 
+/* Disable the selected endpoint */
+int usb_dc_ep_disable(u8_t ep)
+{
+	UsbDevice *regs = &REGS->DEVICE;
+	u8_t ep_num = ep & ~USB_EP_DIR_MASK;
+	UsbDeviceEndpoint *endpoint = &regs->DeviceEndpoint[ep_num];
+
+	if (ep_num >= USB_NUM_ENDPOINTS) {
+		LOG_ERR("endpoint index/address out of range");
+		return -1;
+	}
+
+	endpoint->EPINTENSET.reg = 0;
```

Comment:
Shouldn't this be:
```suggestion
	endpoint->EPINTENCLR.reg = USB_DEVICE_EPINTENCLR_TRCPT0 |
					   USB_DEVICE_EPINTENCLR_TRCPT1 |
					   USB_DEVICE_EPINTENCLR_RXSTP;
```

Since writing 0 to `EPINTENSET` has (should have? I haven't tested) no effect.